articles/tutorials/get-user-information-with-unbounce-landing-pages.md

```diff
@@ -21,13 +21,13 @@ description: How to get user information with one-click social authentication on
 * Add a new JavaScript to your Unbounce landing page, select `Before Body End Tag` under `Placement` and add this code. Also make sure to check jQuery as a dependency.
 
 ```
-<script src="${auth0js_url}"></script>
+<script src="${auth0js_urlv8}"></script>
 <script type="application/javascript">
 
-  var auth0 = new Auth0({
+  var webAuth = new auth0.WebAuth({
     domain:                 '${account.namespace}',
     clientID:               '${account.clientId}',
-    callbackURL:            'REPLACE_WITH_YOUR_UNBOUNCE_PAGE_URL', // e.g http://unbouncepages.com/changeit
+    redirectUri:            'REPLACE_WITH_YOUR_UNBOUNCE_PAGE_URL', // e.g http://unbouncepages.com/changeit
     responseType: 'token'
```

Comment:
@angelaLazar This doesn't look complete. For example, this script is part of this doc https://auth0-docs-content-pr-3675.herokuapp.com/docs/connections/passwordless/spa-sms#use-your-own-ui but if you see the next scripts have not been updated. Also, this doc is accompanied by a sample https://github.com/auth0-samples/auth0-jquery-passwordless-sample which still uses auth0.js 6 (https://github.com/auth0-samples/auth0-jquery-passwordless-sample/blob/master/custom-sms.html#L47) We cannot have the doc using one version and the sample using another, it's inconsistent and confusing to customers.

@jeffreylees can you follow up with someone for this? if we want to update the passwordless scripts (which we should since the API now uses v8) the samples have to be updated as well (or removed from the doc). Will someone from engineering do it?